### DISCUSSION TITLE
Real anchor within tag

### DISCUSSION THREAD

**Discussion by jossmac** [adobe/react-spectrum#7003] [Ideas]
I've implemented a real anchor within a custom `Tag`, so users can right-click and open in new tab, etc. Here's the pseudo-code:

```jsx
<row>
  <gridcell>
    <link tabIndex=-1>contents</link>
    <clearbutton />
  </gridcell>
</row>
```

The clear button is a sibling of the anchor so no invalid html. After some, admittedly limited, testing with VO everything seems to work as expected. Is there any intent to do something similar with react-aria-components, rather than the current synthetic link approach?

**Comment by devongovett** [comment]
A link within a gridcell is fine. Currently, the HTML spec does not allow an `<a>` element itself to have role=gridcell or row however. I raised an issue about this in the past, and there was some discussion about it there: https://github.com/w3c/html-aria/issues/473

**Reply by jossmac** [reply]
Thanks for the link, loads of context!
